main.py

```diff
@@ -1,33 +1,41 @@
-from discord.ext import commands
-import gspread
 import configparser
-import schedule
-import time
-from datetime import datetime
-from oauth2client.service_account import ServiceAccountCredentials
+
+from Commands.IdentityCommand import IdentityCommand
+from Commands.OnyAttunementCommand import OnyAttunementCommand
+from Commands.RaidsCommand import RaidsCommand
+from Commands.RegisterCommand import RegisterCommand
+from Commands.RolesCommand import RolesCommand
+from Commands.SheetCommand import SheetCommand
+
+from DataProviders.GoogleCredentialProvider import get_google_credentials
+from DataProviders.JorachBotProvider import get_jorach
+
+# Global Variables
+
+wow_classes = ["druid", "hunter", "mage", "paladin", "priest", "rogue", "warlock", "warrior"]
```

Comment:
This data should be pulled out into a separate module. It's largely static and can be referred to by importing as necessary. I actually have a PR with a separate module for this stuff that also has mappings for class -> spec -> role type (in that hierarchical order).

Reply:
e.g. if you specify that you're Holy Priest, it automagically infers that you are a healer.